atomic_reactor/constants.py

```diff
@@ -189,3 +192,21 @@
 
 # location for build info directory in the image
 IMAGE_BUILD_INFO_DIR = '/root/buildinfo/'
+
+
+DistgitConfigFile = namedtuple('DistgitConfigFile', ['filename', 'schema'])
```

Comment:
`DistGit` is implementation detail, let's call it more generic `OSBSConfigFile`, `GitConfigFile` or `UserConfigFile`, or something like that.

Reply:
or `InRepoConfigFile`